docs/blogs/zoda.html

```diff
@@ -0,0 +1,387 @@
+        has all of the data, so you have more information than required
+        on the network. If one node had one half of the data, and some
+        other node the other half, you would still be able to recover
+        it, by combining their halves.</p>
+        <p>We could extend the logic further still: each of the <span
+        class="math inline">m</span> participants could hold <span
+        class="math inline">\frac{1}{m}</span> of the data. All
+        together, the participants hold it all, distributed as thinly as
+        possible.</p>
+        <p>In this case, the leader’s transmission cost is now just
+        <span class="math inline">m \cdot \frac{D}{m}</span> = D$, quite
+        the improvement. If the participants want to recover the whole
+        data, each of them will need to send their shard to the others,
+        at a cost of <span class="math inline">(m - 1) \cdot
+        \frac{D}{m}</span> bytes. The total amount of data sent by the
+        leader, and then the followers, is <span class="math inline">D +
+        m \cdot \frac{(m - 1) D}{m} = m \cdot D</span>. Compared to the
+        naive approach, the total amount of data sent is the same.
+        However, it is sent much more efficiently, spreading the load
+        evenly across all the links in the network. Each node sending
+        <span class="math inline">\frac{(m - 1)}{m} \cdot D</span> bytes
+        worth of data to others can operate in parallel, whereas in the
+        original case, the leader must send <span class="math inline">m
+        \cdot D</span> bytes sequentially.</p>
+        <h3 id="dealing-with-loss">Dealing with Loss</h3>
+        <p>One flaw in our scheme, so far, is that the data is spread so
+        thinly, that even one node missing out results in it being lost
+        completely. This might happen by accident—a node crashing is not
+        impossible, after all— but we also want to tolerate malicious
+        nodes. One follower being able to block transmission is not
+        acceptable.</p>
+        <p>From here, we reach to using a <em>coding scheme</em>. This
+        scheme takes in a message of <span class="math inline">n</span>
+        <em>symbols</em>—we shall revisit the term, but think of it like
+        a byte, or some other small piece of data—and produce <span
+        class="math inline">m \geq n</span> symbols. A useful coding
+        scheme has the property that given any <span
+        class="math inline">n</span> of these <span
+        class="math inline">m</span> symbols, we can recover the
+        original message.</p>
+        <p>As an example, a <em>Reed-Solomon</em> code consists of
+        treating the data as a list of <span
+        class="math inline">n</span> field elements (serving as the
+        aforementioned symbols) which define the polynomial:</p>
+        <p><span class="math display">
+        d(X) \coloneqq a_0 + a_1 X + \cdots + a_{n - 1} X^{n - 1}
```

Comment:
It doesn't look like this math formatting works: https://96bc334d.monorepo-eu0.pages.dev/blogs/zoda

<img width="1592" height="312" alt="image" src="https://github.com/user-attachments/assets/8b344c94-5a7a-443f-bcc5-6963bdb80306" />